packages/ember-glimmer/lib/helpers/component.ts

```diff
@@ -0,0 +1,135 @@
+/**
+  @module ember
+*/
+
+/**
+  The `{{component}}` helper lets you add instances of `Component` to a
+  template. See [Component](/api/classes/Ember.Component.html) for
```

Comment:
can we modify this link to link it to the new docs url?

Reply:
OK. What is the new url?

Reply:
https://emberjs.com/api/ember/release/classes/Component

Reply:
thanks!